pages/linux/openvpn3.md

```diff
@@ -0,0 +1,24 @@
+# openvpn3
+
+> OpenVPN 3 Linux client.
+> More information: <https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.
+
+- Start new VPN session:
+
+`openvpn3 session-start --config {{path/to/config.conf}}`
+
+- List established sessions:
+
+`openvpn3 sessions-list`
+
+- Disconnect established session started with given configuration:
```

Comment:
```suggestion
- Disconnect the currently established session started with given configuration:
```

Reply:
```suggestion
- Disconnect the current established session started with given configuration:
```

Reply:
not quite sure which is correct gramatically since there are 2 suggestions but the first suggestion sounds better to me so went with that. happy to change if that is incorrect:

```
- Disconnect the currently established session started with given configuration:
```